### ISSUE TITLE
May I ask if it is possible to use dual encoders and dual classifiers for accent conversion

### ISSUE THREAD

**Issue by hhhuazi** [yl4579/StarGANv2-VC#88]
May I ask if it is possible to use dual encoders and dual classifiers for accent conversion

**Comment by yl4579** [comment]
Yes, as long as you make the domain different accents and use a good ASR model like large speech langauge models (see our latest work: https://arxiv.org/abs/2307.09435). In my experience this framework is very versatile and works for various conversion tasks like accent and emotion conversion. Let me know how it works for you.

**Comment by annabeth97c** [comment]
I've been trying to use variations of this for accent conversion, without much success yet. Haven't tried multiple encoders yet! Thanks for your comment, will try it out

**Comment by hhhuazi** [comment]
> I've been trying to use variations of this for accent conversion, without much success yet. Haven't tried multiple encoders yet! Thanks for your comment, will try it out

Have you used multiple encoders? How is the effect

**Comment by annabeth97c** [comment]
I tried, but I did not get any positive results so far. Originally I was trying with American and British accent where the differences are more subtle. Then I tried with American and Indian accent where the difference is more drastic. I also complemented the data with speaker transferred samples, to allow for cross labelled samples. None of the approaches have resulted in any meaningful feature extraction so far
